rotary/repair.py

```diff
@@ -213,7 +211,6 @@ def map_long_reads(contig_filepath: str, long_read_filepath: str, output_bam_fil
 
     with open(log_filepath, write_mode) as logfile_handle:
         with open(output_bam_filepath, 'w') as bam_handle:
-
             # TODO - add support for different flags like -ax for pacbio
             minimap_args = [dependency_dict['minimap2'], '-t', str(threads), '-ax', 'map-ont', contig_filepath,
                             long_read_filepath]
```

Comment:
Many of your ```subprocess.run``` calls have similar code. They could be refactored into a function that takes the CLI argument string/list and the error logging file handles and then runs ```subprocess.run```  inside. For example, if this was already refactored this way we would have only one place to add ```shell=true```.

Reply:
@LeeBergstrand There are enough variations within the `subprocess.run` calls (e.g., where piping is used, like in `map_long_reads`, or where STDERR is pushed to STDOUT in `run_circlator_merge`, or where the program is allowed to fail in `run_flye`) that I felt it was better to not summarize these `subprocess.run` calls in a single function. I was concerned that a summary function would need to have lots of arguments (and could be confusing to understand) in order to meet the needs of each of the invocations of `subprocess.run`. But maybe it would not be as bad as I imagine? Let me know your thoughts.

Reply:
@jmtsuji I just created https://github.com/jmtsuji/rotary/pull/66 to do the above refactoring. Minor edits could be made. For example, moving the ```logger debug``` into a function with the ```subprocess.run``` call so the log is not called multiple times. This way, you can change the debug logger statement without changing all eight of them manually.

Reply:
Thanks -- I reviewed and have approved the current version. I have to run to a (likely) all-day meeting now, so I cannot look into details further, but feel free to continue to work on this today if you would like.

Reply:
I think I've covered all other needed code changes, so I am also OK with you merging (or approving merge) of this PR if you think it's ready enough for now.

Reply:
(Or feel free to leave further TODOs on my end)

Reply:
I'll resolve this conversation now that #66 is merged.